packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -769,6 +769,19 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
         return (currentActiveRequests() && getVotePhase() == Phase.Reveal);
     }
 
+    function _computePendingStakes(address wallet, uint256 amount) internal override {
+        if (getVotePhase() != Phase.Reveal) return;
+        uint256 currentRoundId = getCurrentRoundId();
+        // If we are during an active reveal phase then we store the pending stake for the roundId
+        for (uint256 i = 0; i < pendingPriceRequests.length; i = unsafe_inc(i)) {
+            if (_getRequestStatus(priceRequests[pendingPriceRequests[i]], currentRoundId) == RequestStatus.Active) {
+                _freezeRoundVariables(currentRoundId);
```

Comment:
the reason you freeze here is pretty nuanced. can you add a comment to this effect?

Reply:
You are right, I've added more comments!